django/db/migrations/operations/models.py

```
@@ -8,9 +8,11 @@
 from .fields import AddField, AlterField, FieldOperation, RemoveField, RenameField
 
 
-def _check_for_duplicates(arg_name, objs):
+def _check_for_duplicates(arg_name, objs, key_word):
     used_vals = set()
     for val in objs:
+        if val in key_words:
```

Ruff (0.8.2)

Undefined name `key_words`

(F821)